portal/server/controllers/utilisation-report-service/utilisation-report-upload/utilisation-report-status.js

```diff
@@ -33,7 +33,7 @@ const getReportPeriod = () => {
  */
 const getReportDueDate = async (userToken, reportPeriodDate = subMonths(new Date(), 1)) => {
   const bankHolidays = await getBankHolidays(userToken);
-  const reportDueDate = getBusinessDayOfMonth(reportPeriodDate, bankHolidays, 10);
+  const reportDueDate = getBusinessDayOfMonth(addMonths(reportPeriodDate, 1), bankHolidays, 10);
```

Comment:
in JSDoc we should mark [optional params with square brackets](https://jsdoc.app/tags-param). i.e.
```
@param {Date} [reportPeriodDate]
```